.github/workflows/test.yaml

```diff
@@ -28,18 +30,13 @@ jobs:
         with:
           fetch-depth: 0
```

Comment:
why do we need whole git history here?

Reply:
Supposedly it's related to dfad24dbc6119897b98c44785a127dc6fee4644b but it's kinda unnecessary in the `test` workflow anyway. Will remove that.